C#_Assignment1_programfiles/Program2.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2_assign1_Q2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+             
+            Customer obj = new Customer();
```

Comment:
name of the object should be self explanatory

Reply:
Changed object names